persistence/test/block_test.go

```diff
@@ -10,16 +10,16 @@ func TestGetBlockStateHash(t *testing.T) {
 	db := NewTestPostgresContext(t, 0)
 
 	// Cannot get prev hash at height 0
-	appHash, err := db.GetBlockHash(0)
+	stateHash, err := db.GetBlockHash(0)
```

Comment:
I am not sure if it's intended but as basically a newbie in this part of the code, I find it "odd" even if it makes sense to see

```go
stateHash, err := db.GetBlockHash()
```
shouldn't this be 

```go
blockHash, err := db.GetBlockHash()
```
?

Reply:
**tl;dr Yes**

Yea, the `appHash` and the `stateHash` are the same thing which was an inconsistency.

I still have to do a bit of research whether we are going to do anything different with `blockHash`. For example, the `quorumCert` and `proposer` are included in the block but not part of the state hash, but TBD.

Reply:
Potentially addressed here: [`66ed7fc` (#391)](https://github.com/pokt-network/pocket/pull/391/commits/66ed7fc307771c9a57e1c31e21ee285984b471bf)

Reply:
Awesome, thank you! 🙏

Reply:
agreed, as my previous comment.